### PR TITLE
Tsi exposev2features

### PR DESCRIPTION
**Pull request by bvanderlugt**
Adding Time Series Insights 2018-08-15-preview

### CODE REVIEW

File: schemas/2015-01-01/deploymentTemplate.json [hunk 1]
```diff
@@ -667,6 +667,10 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-11-15/Microsoft.TimeSeriesInsights.json#/resourceDefinitions/environments_eventSources" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-11-15/Microsoft.TimeSeriesInsights.json#/resourceDefinitions/environments_referenceDataSets" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-11-15/Microsoft.TimeSeriesInsights.json#/resourceDefinitions/environments_accessPolicies" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-08-15-preview/Microsoft.TimeSeriesInsights.json#/resourceDefinitions/environments" },
```
Comment (TianoMS) [inline schemas/2015-01-01/deploymentTemplate.json]: >{ "$ref": "https://schema.management.azure.com/schemas/2018-08-15-preview/Microsoft.TimeSeriesInsights.json#/resourceDefinitions/environments" }, [](start = 18, length = 145)

we have a new root schema now: 2019-04-01/deploymentTemplate.json. can you please add them to the new root schema as well?

Comment (bvanderlugt) [reply schemas/2015-01-01/deploymentTemplate.json]: Added

File: tools/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,745 @@
+{
```
Comment (Tiano2017) [inline tools/package-lock.json]: do we need this file?

Comment (bvanderlugt) [reply tools/package-lock.json]: I removed package-lock.

You might want to consider adding it to you .gitignore

**Pull request merged**